package.json

```diff
@@ -22,7 +22,8 @@
     "release:pack": "tsx scripts/releasePack.mts",
     "docs:api": "rimraf --glob ./docs/pages/**/api-docs ./docs/pages/**/api && pnpm docs:api:build",
     "docs:api:build": "tsx ./scripts/buidApiDocs/index.ts",
-    "docs:build": "pnpm --filter docs build",
+    "docs:llms:build": "rimraf --glob ./docs/public/material-ui/ && tsx ./scripts/buildLlmsDocs/index.ts --projectSettings ./packages/api-docs-builder-core/materialUi/projectSettings.ts --nonComponentFolders material/getting-started material/customization material/experimental-api material/guides material/integrations material/migration",
```

Comment:
Could we add `nonComponentFolders` as a list on the `projectSettings` to avoid having to pass a long list of folders in the command?